package.json

```diff
@@ -6,7 +6,8 @@
   "scripts": {
     "build": "tsc",
     "start": "npm run build && node ./dist/app.js",
-    "lint": "npx @biomejs/biome check --write *.ts listeners",
+    "lint": "npx @biomejs/biome check *.ts listeners",
+    "lint:fix": "npx @biomejs/biome check --write *.ts listeners",
```

Comment:
No reference to `npm run lint` in the `README` which I think is alright, but I'm unsure if the `:` approach makes sense here 🤔

Reply:
Adding `npm run lint:ci` and using it for CI jobs may be another option, but I'm happy to let @filmaj and you decide the naming tomorrow!

Reply:
Either works for me 👍

Reply:
> Adding `npm run lint:ci` and using it for CI jobs may be another option

@seratch Thank you for the suggestion and apologies for a slow response 😓

I like `lint:ci` but when making that change I found an update to `npm run test` which was somewhat unexpected when testing other code changes.

I do think an `npm run ci` job might be neat, but right now we're using `npm run test` to check `build` and `lint` which I think covers this 👀